script/start.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+set -x
+set -e
+
+mlflow db upgrade "$JOBSERVER_RDB_URL"
+
+mlflow server \
+    --backend-store-uri "$JOBSERVER_RDB_URL" \
+    --default-artifact-root "s3://grnds-$AWS_ENVIRONMENT-cortex-shared/mlflow" \
```

Comment:
If we want to use MLFlow in other non-cortex projects (e.g., for NLP) would we want to use a bucket name that doesn't have `cortex` in it?

Reply:
We could, but I don't think it really matters either. And we would still have to put the terraform somewhere which would probably live in the cortex repo, so it seems inescapable.